src/features/content-answers/ContentAnswersOpenButton.scss

```diff
@@ -17,4 +31,28 @@
     &:not(.bdl-is-disabled):hover {
         background-color: $bdl-gray-10;
     }
+
+    &.bdl-ContentAnswersOpenButton--hasQuestions {
+        @include BackgroundGradient;
+
+        .bdl-BoxAIIcon {
+            path {
+                fill: $white;
+            }
+        }
+    }
+
+    &.bdl-ContentAnswersOpenButton--hasQuestions:hover {
+        @include BackgroundGradientHover;
+    }
+
+    &.bdl-ContentAnswersOpenButton--hasQuestions:not(.bdl-is-disabled):focus,
+    &.bdl-ContentAnswersOpenButton--hasQuestions:not(.is-disabled):focus {
```

Comment:
the eventual goal is to replace `is-disabled` with `bdl-is-disabled` so it shouldn't need both. I think you can remove the second line